### PR TITLE
Improve MonitorBusinessCritical failure message

### PR DESCRIPTION
**Pull request by joofsh**
Improve the text formatting of the failure message for the MonitorBusinessCriticalJobsJob

- Declutters the output, no longer repeating the "last started at" when failure has happened
- Remove double "@here". Now only one will happen

connects #3023 

screen shot 2017-08-25 at 6 11 15 pm

### CODE REVIEW

File: app/services/prometheus_service.rb [hunk 1]
```diff
@@ -127,6 +127,7 @@ def push_metrics!
       metrics = Prometheus::Client.registry
       url = Rails.application.secrets.prometheus_push_gateway_url
 
+      return unless url
```
Comment (joofsh) [inline app/services/prometheus_service.rb]: unrelated change. But this was causing errors when running sidekiq locally if you don't also run a push gateway. I'm guessing no one uses sidekiq locally!

Comment (kierachell) [reply app/services/prometheus_service.rb]: I tried; but I figured it out and just had to "fake push" it.

File: app/jobs/monitor_business_critical_jobs_job.rb [hunk 2]
```diff
@@ -11,7 +11,7 @@ class MonitorBusinessCriticalJobsJob < CaseflowJob
 
   ALERT_THRESHOLD_IN_HOURS = 5 # in hours
```
Comment (kierachell) [inline app/jobs/monitor_business_critical_jobs_job.rb]: How much of a PIA would it be to have this as a env or redis config?

**Comment by kierachell** [comment]
Also, can you enable this in preprod?
